.github/workflows/operator-ci.yaml

```diff
@@ -141,4 +141,24 @@ jobs:
         with:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GOSEC_OUTPUT: "junit-xml:/github/workspace/gosec-results.xml"
-          
\ No newline at end of file
+
+  e2e:
```

Comment:
I would like you to make another separate workflow for this particular test-case. 
workflow name could be `e2e-test.yaml`